### DISCUSSION TITLE
第5节课笔记(5班)

### DISCUSSION THREAD

**Discussion by sanbuphy** [InternLM/Tutorial#328] [笔记]
以下上传宝们的第五节课笔记

**Comment by ZhaoHongSen0726** [comment]
https://blog.csdn.net/m0_75085274/article/details/135579719?spm=1001.2014.3001.5501

**Comment by DreamBinary** [comment]
https://github.com/DreamBinary/InternLMCamp/tree/master/day05/note

**Comment by GitEasonXu** [comment]
https://github.com/GitEasonXu/InternLM/blob/main/homework/course_5/coursenote.md

**Reply by GitEasonXu** [reply]
加油

**Comment by shengmouget** [comment]
https://blog.csdn.net/shengweiit/article/details/135565624

**Comment by JayceNing** [comment]
https://zhuanlan.zhihu.com/p/678159563

**Comment by angellyao** [comment]
https://zhuanlan.zhihu.com/p/678433976

**Comment by Milkve** [comment]
[第五节笔记](https://juejin.cn/spost/7325242594651848740)

**Comment by ATang0729** [comment]
https://github.com/ATang0729/InternLMCamp_Notes/blob/main/notes/class5-note/class5_note.md
